package.json

```diff
@@ -17,8 +17,8 @@
     "tdexd:run": "bash scripts/tdexd-run",
     "tdexd:run-initialized": "bash scripts/tdexd-run-initialized",
     "tdexd:tdexdconnect": " docker exec -it tdexd tdexdconnect --out url",
-    "tauri:build": "npx tauri build",
-    "tauri:dev": "npx tauri dev"
+    "tauri:build": "cargo tauri build",
+    "tauri:dev": "cargo tauri dev"
```

Comment:
I get the following error after this change:

```bash
$ cargo tauri dev
# error: no such subcommand: `tauri`
```

Reply:
Yeah, you need to install the CLI with cargo, as I said in the decscription I used that from rust instead the one as npm package since it breaks cross installation on docker container

Reply:
`cargo install tauri-cli`